src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReferenceConfidenceModel.java

```diff
@@ -193,18 +193,19 @@ public ReferenceConfidenceModel(final SampleList samples,
         final String sampleName = readLikelihoods.getSample(0);
 
         final int globalRefOffset = refSpan.getStart() - activeRegion.getExtendedSpan().getStart();
-        for ( final ReadPileup pileup : refPileups ) {
+        final int refPileupsSize = refPileups.size();
+        for (int i = 0; i < refPileupsSize; i++) {
```

Comment:
Add comment here that the old-school for loop is intentional (for-each was found to show up on the profiler).